src/Symfony/Component/Cache/Adapter/FilesystemAdapter.php

```diff
@@ -147,7 +147,7 @@ protected function doSave(array $values, $lifetime)
     private function getFile($id, $mkdir = false)
     {
         $hash = str_replace('/', '-', base64_encode(md5($id, true)));
```

Comment:
Whatever the probably is, it just means, you cannot trust cache keys. E.g. it's not safe to use the current request info as cache key. That should be documented.